src/app/types.rs

```diff
@@ -1,12 +1,7 @@
 use super::OrdersContainer;
-use crate::browser::Url;
-use crate::virtual_dom::EventHandler;
 
-pub type UpdateFn<Ms, Mdl, INodes, GMs> =
-    fn(Ms, &mut Mdl, &mut OrdersContainer<Ms, Mdl, INodes, GMs>);
-pub type SinkFn<Ms, Mdl, INodes, GMs> =
-    fn(GMs, &mut Mdl, &mut OrdersContainer<Ms, Mdl, INodes, GMs>);
+// @TODO_B: Move content to more appropriate files and delete this file.
```

Comment:
Still need to think about this TODO a bit.

Reply:
It will be the last step. Don't bother with it now.